src/read/elf/file.rs

```diff
@@ -61,6 +61,26 @@ impl<'data, Elf: FileHeader> ElfFile<'data, Elf> {
         })
     }
 
+    /// Returns the endianness.
+    pub fn endianess(&self) -> Elf::Endian {
```

Comment:
```suggestion
    pub fn endian(&self) -> Elf::Endian {
```

Let's keep this a different name from `Object::endianness`.